src/components/EventTypeFilters.vue

```diff
@@ -1,12 +1,22 @@
 <template>
-  <div class="event-type-filters">
-    <h5 class="event-filters-title">Type of event</h5>
-    <div v-for="(label, type) in eventTypes">
-      <input type="checkbox"
-        class="no-margin-checkbox"
-        :checked="selectedEventTypes.includes(type)" 
-        @change="updateSelectedEventTypes(type, $event)"
-      /> {{label}}
+  <div>
+    <h5 v-if="showTitle" class="filter-events-title">Type of event</h5>
+    <div 
+      class="filter-events-item" 
+      v-for="(label, type) in eventTypes"
+    >
+      <div class="filter-events-checkbox">
```

Comment:
Hm never heard of such a limitation, curious to learn more about it if you stumble on it. :)